src/mkieee.f

```diff
@@ -59,7 +59,7 @@ subroutine mkieee(a,rieee,num)
               n = n - 1
            enddo
         endif
-!        n=floor(alog(atemp)/alog2)
+        n=floor(alog(atemp)/alog2)
```

Comment:
Is this change unintentional? You should not need to change the code, just the tests.

Reply:
If you can remove this change we can merge the rest of the PR and get these tests into use...

Reply:
ok sounds good